### ISSUE TITLE
Using twitter.cfg file

### ISSUE THREAD

**Issue by jayashreeallamshetty** [iit-cs579/main#483]
The a0.py file prompts for twitter credentials. should we add the twitter credentials there(which the professor said is not the right practice during the lecture) or should we have it as a separate .cfg file for the assignment?

**Comment by aronwc** [comment]
Yes, please add your credentials to a0.py directly. This is not good practice, but makes it more convenient for grading. Your repo will be private, so only you and I will see them.
